webhook.sh

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+set -x
+echo excute azure webhook to trigger restart with latest docker image
+
+curl -dH -X POST "https://\$$WEBHOOK_USERNAME:$WEBHOOK_PASSWORD@pdm-todo.scm.azurewebsites.net/docker/hook"
```

Comment:
```suggestion
curl -dH --fail -X POST "https://\$$WEBHOOK_USERNAME:$WEBHOOK_PASSWORD@pdm-todo.scm.azurewebsites.net/docker/hook"
```
Curl doesn't normally consider 4xx or 5xx responses errors, but we do want the pipeline to fail if that happens.